build-logic-settings/gradle.properties

```diff
@@ -20,3 +20,6 @@ org.gradle.caching=true
 # from common properties
 # https://github.com/gradle/gradle/issues/4823
 org.gradle.configureondemand=false
+# from common properties
+# Limited locally, because unlimited workers will hang pc on parallel gradleTest execution
+org.gradle.workers.max=4
```

Comment:
FYI i override this in my local gradle.properties
And I don't sure that you set the optimal value
I want to replace that or add a comment on how to choose the correct number for my local machine

Reply:
yep, anyone could override
I don't know yet how to choose a proper value, i tried to set a viable default, like i said in PR description it is tested on problematic scenario: run gradle_tests

Reply:
macbook with 4 workers:
./gradlew -p subprojects gradleTest
BUILD SUCCESSFUL in 9m 25s

without noticeable ide lags while building